app/models/concerns/user_totp_methods.rb

```diff
@@ -37,8 +36,8 @@ def enable_totp!(seed, level)
 
     if mfa_device_count_one?
       self.mfa_level = level
-      self.mfa_recovery_codes = Array.new(10).map { SecureRandom.hex(6) }
-      self.mfa_hashed_recovery_codes = mfa_recovery_codes.map { |code| BCrypt::Password.create(code) }
+      self.new_mfa_recovery_codes = Array.new(10).map { SecureRandom.hex(6) }
+      self.mfa_hashed_recovery_codes = new_mfa_recovery_codes.map { |code| BCrypt::Password.create(code) }
```

Comment:
Same with this, should we move this to a method in user multifactor methods (webauthn credentials also has this same logic)?

Reply:
done!